src/models/interfaces/IActions.ts

```diff
@@ -3,13 +3,14 @@ export const KeyboardPlatforms = [
 	'Fire TV',
 	'Roku',
 	'Kodi',
+	'Sony BRAVIA'
```

Comment:
Since `Sony BRAVIA` is a keyboard platform, it should also be added to the switch cases in [`keyboard-dialog.ts`](https://github.com/Nerwyn/android-tv-card/blob/main/src/classes/keyboard-dialog.ts) as well. `Android TV` uses a mix of ADB and the Android TV remote API, and `Fire TV` uses purely ADB. Can you add it to the appropriate cases? I imagine it needs purely ADB (Fire TV) but you should do some tests.